tests/installation/partitioning_raid.pm

```diff
@@ -263,10 +284,16 @@ sub run() {
     if (check_screen 'subvolumes-shadowed', 5) {
         send_key 'alt-y';
     }
-
-    if (get_var("LVM")) {
+    # check overview page for Suggested partitioning
+    if (get_var("LVM") and !get_var("UEFI")) {
```

Comment:
Thanks! I see your point, I just wanted to show all possible combinations without a need to parse nested conditions in reviewer's mind :-) If you really insist on that I can modify.